### PR TITLE
Fix broken link on robots.txt

### PR DESCRIPTION
**Pull request by coliff**

### CODE REVIEW

File: public/robots.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# See http://www.robotstxt.org/wc/norobots.html for documentation on how to use the robots.txt file
+# See https://www.robotstxt.org/ for documentation on how to use the robots.txt file
```
Comment (peff) [inline public/robots.txt:1]: Should this remain `http://`? They seem to issue a redirect back on the https port (which seems like a curious thing to bother setting up, but whatever...).

**Comment by peff** [comment]
Should we perhaps just get rid of this `robots.txt` file entirely? It doesn't have any uncommented lines.

**Comment by coliff** [comment]
overall, I think it's better to have an empty `robots.txt` than none at all since search engines will look for it here automatically and if none is here it'd result in a 404.

**Comment by peff** [comment]
Fair enough. Why don't we just make it an empty file, then?

**Comment by coliff** [comment]
you can edit the PR to remove the comments if you want, but I think it's fine it keep it.

**Pull request merged**